covid_model/model_specs.py

```diff
@@ -65,14 +65,14 @@ def from_db(cls, engine, spec_id, new_end_date=None):
         specs.spec_id = row['spec_id']
         specs.base_spec_id = row['base_spec_id']
 
-        specs.set_tc(tslices=row['tslices'], tc=row['tc'], tc_cov=row['tc_cov'])
-        specs.set_model_params(row['model_params'])
+        specs.set_tc(tslices=row['tslices'], tc=row['tc'], tc_cov=json.loads(row['tc_cov'].replace('{', '[').replace('}', ']')))
```

Comment:
due to change in how bigquery returns queries?

Reply:
In their data type options, to be specific. Turns out BigQuery doesn't support dynamic JSON structures or arrays of arrays, so those are stored as strings. They have a JSON type in beta, but for now it's fine --- we can just parse the strings.